### DISCUSSION TITLE
Question with Favicon

### DISCUSSION THREAD

**Discussion by crashzk** [ianlucas/cs2-inventory-simulator#142] [General]
### `appFaviconUrl`

- URL of the app's favicon. If empty, uses Inventory Simulator's favicon.
- **Type:** `string`
- **Default:** _Empty_

### `appFaviconMimeType`

- MIME type of the app's favicon. If empty, uses Inventory Simulator's favicon MIME type.
- **Type:** `string`
- **Default:** _Empty_

How does the link to the images work?

https://cdn.discordapp.com/attachments/672618501042733076/1320140465189945344/Logo.png?ex=6768841d&is=6767329d&hm=75ac64dcbbbc44a9c6de378a2b96d7a3aa00c0cf7d09af4dd095e889db340c7b&

Create this in PNG but even informing the same link for both options the Favicon does not change.

**Comment by ianlucas** [comment]
Ideally `appFaviconUrl` should point to a private CDN or a place that you know for sure won't get the image deleted. I'd not recommend using Discord's CDN. Since you have a fork, you can place the favicon in `public` folder, and update `appFaviconUrl` to `/favicon.png` and `appFaviconMimeType` to `image/png`.
